apps/meeting/src/components/DeviceSelection/CameraDevices/index.tsx

```diff
@@ -8,15 +8,16 @@ import {
   QualitySelection,
   CameraSelection,
   Label,
-  BackgroundBlurCheckbox,
 } from 'amazon-chime-sdk-component-library-react';
 
 import { title, StyledInputGroup } from '../Styled';
 import { useAppState } from '../../../providers/AppStateProvider';
-import { BlurValues } from '../../../types';
+import { VideoFiltersCpuUtilization } from '../../../types';
+import { VideoTransformDropdown } from '../CameraDevices/VideoTransformDropdown';
 
 const CameraDevices = () => {
-  const { blurOption } = useAppState();
+  const { videoTransformCpuUtilization } = useAppState();
+  const videoTransformsEnabled = videoTransformCpuUtilization != VideoFiltersCpuUtilization.Disabled;
```

Comment:
Use strict comparison `!==` and `===` because using non-strict equality is bug prone. (e.g., `false != 0` is `false` in JS).

Please fix other files too in this PR.

Reply:
done